go/worker/storage/committee/node.go

```diff
@@ -627,6 +711,11 @@ func (n *Node) worker() { // nolint: gocyclo
 		return
 	}
 
+	// Start runtime node watcher.
+	go n.runtimeNodesWatcher()
+	// Start the quit watcher.
+	go n.watchQuit()
```

Comment:
Isn't this something that should be spawned in `Start()`?

Reply:
yeah agreed, makes more sense spawning there